shared/common-adapters/button-bar.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import * as React from 'react'
+import Box from './box'
+import {globalStyles, isMobile} from '../styles'
+
+type Props = {
+  direction: 'row' | 'column',
+  align?: 'flex-start' | 'flex-end' | 'center', // ignored by column
+  children: React.Node,
+  fullWidth?: boolean, // ignored by column
+  small?: boolean, // ignored by column
+  style?: any,
+}
+
+class ButtonBar extends React.PureComponent<Props> {
+  static defaultProps = {
+    align: 'center',
+    direction: 'row',
+    fullWidth: false,
+    small: false,
+  }
+
+  _spacing = () => {
+    if (this.props.direction === 'row' && this.props.small && !isMobile) {
+      return SmallSpacer
+    }
+
+    return BigSpacer
+  }
+
+  _surroundSpacing = () => {
+    return this.props.direction === 'column'
+  }
+
+  render() {
+    const Spacing = this._spacing()
+    const surroundSpacing = this._surroundSpacing()
+    const children = React.Children.toArray(this.props.children)
+    const childrenWithSpacing = children.reduce((arr, c, idx) => {
```

Comment:
🐑  🐑 🐑 You can add a `React.Children.toArray` here too. That way you don't have to worry about the key.